schema/attribute/national-id.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Why is this split into two types? We can have one attribute that references two images - one for front and one for back.

Reply:
historical, will look into merging it